src/Admin/ElectedRepresentativeAdmin.php

```diff
@@ -3,8 +3,12 @@
 namespace AppBundle\Admin;
 
 use AppBundle\Address\Address;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\ElectedRepresentative\ElectedRepresentative;
+use AppBundle\Entity\ElectedRepresentative\Label;
 use AppBundle\Form\AdherentEmailType;
 use AppBundle\Form\GenderType;
+use AppBundle\Form\LabelType;
```

Comment:
`LabelType` is too generic, what about `ElectedRepresentativeLabelType`?

Reply:
@ottaviano in the end of développement I think to create a folder `ElectedRepresentative` for all form types. It will be a good alternative, will not it?